iFixit/react-commerce
# Allow hiding children from product lists parents

🚫 deploy_blocked waiting for #1624

closes #1406 

This PR hide product list children from their parent collection according to their `hideFromParent` flag.

### QA

1. Visit [Vercel preview](https://react-commerce-git-allow-hiding-children-from-pro-4cf252-ifixit.vercel.app/Tools)
2. Try to change the `hideFromParent` flag value
3. Verify that children are correctly hidden from the product list children section when their flag is set to true
4. Verify that the breadcrumbs in the child collection are still correctly referencing the parent collection
